### ISSUE TITLE
Focus visible understanding doc references removed/modified SC

### ISSUE THREAD

**Issue by backwardok** [w3c/wcag#3696]
The [intent section of the focus visible understanding doc](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html#intent) includes the following:

> New in WCAG 22: This criterion does not specify what the form is, but [Focus Appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum) does define how visible the indicator should be. Passing [Focus Appearance (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum) would pass this success criterion.

However, the SC that it's pointing to (Focus not obscured) does not define how visible the indicator should be. My understanding from some of the earlier versions of WCAG 2.2 was that there was an SC that was being worked on called Focus Appearance, but that didn't make the final cut of WCAG 2.2.

This part of the understanding doc should probably get updated or be removed.

**Comment by backwardok** [comment]
ah looks like this is a duplicate of https://github.com/w3c/wcag/issues/3696
